src/service/OpsLevelController.ts

```diff
@@ -119,8 +119,8 @@ export class OpsLevelController {
         timeout: { hours: 2 },
         initialDelay: { seconds: 10 },
         id: "opslevel-exporter",
-        fn: async (abortSignal: AbortSignal) => { await this.exportToOpsLevel(abortSignal) },
-        signal: this.running_task_abort_controller.signal,
+        fn: async (abortSignal: any) => { await this.exportToOpsLevel(abortSignal) },
+        signal: (this.running_task_abort_controller.signal as any),
```

Comment:
Why was this change made?

Reply:
because I was using the wrong abortcontroller lol. fixed.